apps/explorer/lib/explorer/chain/celo/celo_election_rewards.ex

```diff
@@ -70,6 +73,57 @@ defmodule Explorer.Chain.CeloElectionRewards do
     )
   end
 
+  def base_aggregated_block_query(block_number, reward_types) do
+    {:ok, zero_wei} = Wei.cast(0)
```

Comment:
persistently declaring `zero_wei` at the top of various methods feels like an anti pattern. We can construct queries via

```elixir
def where_non_zero_reward(query), do: query |> where([q], q.amount != %Wei{value: 0})
```